python/pyspark/sql/session.py

```diff
@@ -39,6 +40,7 @@
 )
 
 from pyspark.conf import SparkConf
+from pyspark.core.files import SparkFiles
```

Comment:
Let's import this within `addArtifacts`. Pure Python package does not have `core` package.